content/doc/book/getting-started/installing.adoc

```diff
@@ -102,58 +102,119 @@ Here, "8081" was chosen but you can put another port available.
 ====
 
 ==== OpenIndiana/Hipster
-On a system running the OpenIndiana/Hipster you can install Jenkins in either local or global zone using the IPS packaging system.
+
+On a system running link:http://www.openindiana.org/[OpenIndiana/Hipster]
+Jenkins can be installed in either the
+local or global zone using the
+link:https://en.wikipedia.org/wiki/Image_Packaging_System[Image Packaging
+System] (IPS).
 
 For the common case of running the newest packaged weekly build as a standalone (Jetty) server, simply execute:
+
 [source,bash]
 ----
 pkg install jenkins
 svcadm enable jenkins
 ----
 
-The common packaging integration for standalone service will:
+The common packaging integration for a standalone service will:
 
 * Create a `jenkins` user to run the service and to own the directory structures under `/var/lib/jenkins`.
-* Pull the OpenJDK8 and other packages required to execute Jenkins, including the `jenkins-core-weekly` package with the latest `jenkins.war` (note: Jenkins LTS releases currently do not support OpenZFS-based systems well, so no packaging is provided at this time).
-* Setup Jenkins as an `svc:/network/http:jenkins` SMF service; you need to `enable` it as shown above, if you want the server to be launched on every start of the zone. See `/lib/svc/manifest/network/jenkins-standalone.xml` for more details.
-* The console log output is managed by SMF at `/var/svc/log/network-http:jenkins.log`. Check this file if you are troubleshooting Jenkins, as well as to find the initial-setup generated administrator password.
-* Set Jenkins to listen on port 8080. Access this port with your browser to start configuration.
-* If you want to change some service attributes, such as the environment variables (e.g. a custom `JENKINS_HOME`), the listen port or other aspects of the command-line invokation, you can run:
+* Pull the OpenJDK8 and other packages required to execute Jenkins, including
+  the `jenkins-core-weekly` package with the latest `jenkins.war`.
++
+CAUTION: Long-Term Support (LTS) releases currently do not support OpenZFS-based
+systems, so no packaging is provided at this time.
+* Set up Jenkins as an `svc:/network/http:jenkins` SMF service; which should be
+  enabled as the `svcadm` command above demonstrates.
+* Set Jenkins to listen on port 8080.
+* Configure the log output is managed by SMF at `/var/svc/log/network-http:jenkins.log`.
```

Comment:
This does not quite sound English :) I think `Configure the` can just be dropped: the stdout/stderr log is managed by SMF, period :)